frontend/src/components/Profile.js

```diff
@@ -222,6 +239,18 @@ function Profile(props) {
                   fullWidth
                 />
               </Box>
+              <Box mb={2}>
+                <ToggleButtonGroup
+                  color="primary"
+                  value={units}
+                  exclusive
+                  onChange={handleUnitChange}
+                  aria-label="Platform"
+                >
+                  <ToggleButton value="imperial">Imperial</ToggleButton>
+                  <ToggleButton value="metric">Metric</ToggleButton>
+                </ToggleButtonGroup>
+              </Box>
```

Comment:
Can the label/title of the textinput boxes change to the respective units when toggled? (lbs to Kg, ft to m)

Reply:
Nice idea